dev/source/docs/sitl-native-on-windows.rst

```diff
@@ -29,12 +29,12 @@ Running SITL
       
 #. Open a Cygwin64 terminal from the start menu
 
-#. Navigate to one of the vehicle directories (in this case Copter) and call ``sim_vehicle.py`` to start SITL.  If you get a windows security alert for the the firewall, allow the connection. 
+#. Navigate and call ``sim_vehicle.py`` to start SITL.  If you get a windows security alert for the the firewall, allow the connection. Specify a vehicle type after ``-v`` parameter.
 
    ::
 
-       cd ~/ardupilot/ArduCopter
-       sim_vehicle.py --map --console
+       cd ~/ardupilot/Tools/autotest
+       ./sim_vehicle.py --map --console -v ArduCopter
```

Comment:
```bash
       cd ~/ardupilot
       ./Tools/autotest/sim_vehicle.py  -v ArduCopter --map --console
```

This is my personal preference. Executing SITL will create several files and hiding them away in autotest folder doesn't seem the best option to me.

Reply:
Applyed your propose... It's almost the same as by @peterbarker